### DISCUSSION TITLE
【求助】为什么我的绳网加载图片这么慢

### DISCUSSION THREAD

**Discussion by hjejdjdj** [share121/inter-knot#1389] [其他]
![1722906453283](https://github.com/user-attachments/assets/be92249b-41c4-4332-996d-0ebc0125b48a)
总感觉加速器没啥用，有可能是我不太会用吧。（我是手机版的

**Comment by wwwwsn147** [comment]
电脑只能用Watt Toolkit对Github加速，也很慢

**Comment by Xiongty0722** [comment]
梯子，启动👍

**Comment by Aisyle-001** [comment]
![容器 1@1x](https://github.com/user-attachments/assets/05dd0cd4-de13-4c2d-bb55-fc6d463d2139)

**Reply by VacuolePaoo** [reply]
确实很好用

**Comment by EhuanyingK** [comment]
这绳网怎么还带诈骗的，举办了

**Comment by Yukio4869** [comment]
可恶，又被骗了

**Comment by UsingPhonix** [comment]
我这挺快的，建议用clash

**Comment by Dave-12138** [comment]
可能是你的超距遥感并没有对`private-user-images.githubusercontent.com`生效
